fe/fe-core/src/main/java/com/starrocks/analysis/OutFileClause.java

```diff
@@ -145,44 +142,44 @@ private void analyzeProperties() throws AnalysisException {
                 throw new AnalysisException(PROP_COLUMN_SEPARATOR + " is only for CSV format");
             }
             columnSeparator = properties.get(PROP_COLUMN_SEPARATOR);
-            processedPropKeys.add(PROP_COLUMN_SEPARATOR);
         }
 
         if (properties.containsKey(PROP_LINE_DELIMITER)) {
             if (!isCsvFormat()) {
                 throw new AnalysisException(PROP_LINE_DELIMITER + " is only for CSV format");
             }
             rowDelimiter = properties.get(PROP_LINE_DELIMITER);
-            processedPropKeys.add(PROP_LINE_DELIMITER);
         }
 
         if (properties.containsKey(PROP_MAX_FILE_SIZE)) {
             maxFileSizeBytes = ParseUtil.analyzeDataVolumn(properties.get(PROP_MAX_FILE_SIZE));
             if (maxFileSizeBytes > MAX_FILE_SIZE_BYTES || maxFileSizeBytes < MIN_FILE_SIZE_BYTES) {
                 throw new AnalysisException("max file size should between 5MB and 2GB. Given: " + maxFileSizeBytes);
             }
-            processedPropKeys.add(PROP_MAX_FILE_SIZE);
         }
     }
 
-    private void getBrokerProperties(String filePath, Set<String> processedPropKeys) {
+    private void setBrokerProperties() {
         boolean outfile_without_broker = false;
         if (!properties.containsKey(PROP_BROKER_NAME)) {
             outfile_without_broker = true;
         }
         String brokerName = null;
         if (!outfile_without_broker) {
             brokerName = properties.get(PROP_BROKER_NAME);
-            processedPropKeys.add(PROP_BROKER_NAME);
         }
 
         Map<String, String> brokerProps = Maps.newHashMap();
         Iterator<Map.Entry<String, String>> iter = properties.entrySet().iterator();
         while (iter.hasNext()) {
             Map.Entry<String, String> entry = iter.next();
-            if (entry.getKey().startsWith(BROKER_PROP_PREFIX) && !entry.getKey().equals(PROP_BROKER_NAME)) {
-                brokerProps.put(entry.getKey().substring(BROKER_PROP_PREFIX.length()), entry.getValue());
-                processedPropKeys.add(entry.getKey());
+            if (!entry.getKey().equals(PROP_BROKER_NAME)) {
```

Comment:
Since all properties will be added to the brokerProps, i think no more need to check if it equals to broker.name?

Reply:
fixed